doc/rst/source/text_common.rst_

```diff
@@ -23,7 +23,8 @@ Appendix :ref:`Chart-Octal-Codes-for-Chars` in the GMT Technical Reference and C
 :doc:`gmt.conf` file in order to use the accented characters. Using the
 **-G** or **-W** options, a rectangle underlying the text may be plotted
 (does not work for strings with sub/super scripts, symbols, or composite
-characters, except in paragraph mode (**-M**)).
+characters, except in paragraph mode (**-M**)).  Finally, you may typeset
+Latex expressions provided they are enclosed in @$...@$.
```

Comment:
```suggestion
Latex expressions provided they are enclosed in @$...@$ and both ``latex`` 
and ``dvips`` are installed.
```

Reply:
I've already rewritten that section to mention @[ and <math> so I think I cannot merge this now since it differs from what is in the branch.